components/raftstore/src/store/worker/read.rs

```diff
@@ -353,6 +354,7 @@ impl ReadDelegate {
             txn_ext: peer.txn_ext.clone(),
             read_progress: peer.read_progress.clone(),
             pending_remove: false,
+            flashback_pending_ignore: false,
```

Comment:
unnecessary, we can simply reset the leader lease to None. And check flashback state in `maybe_renew_leader_lease`